docs/ref/django-admin.txt

```diff
@@ -1584,6 +1589,16 @@ it could be useful if you have a ``ForeignKey`` in
 allow creating an instance instead of entering the primary key of an existing
 instance.
 
+.. versionadded:: 3.0
```

Comment:
Let's move this above the `.. django-admin-option:: --database DATABASE` block, such that, when we drop the `versionadded` part, we don't need to rephrase the text.